apps/spruce/src/pages/container/EventsTable/index.tsx

```diff
@@ -79,7 +79,6 @@ const EventsTable: React.FC<{}> = () => {
     defaultColumn: {
       enableColumnFilter: false,
     },
-    manualPagination: true,
```

Comment:
Was this change intentional? It looks like the page is unrelated to the packages table